app/routers/event.py

```diff
@@ -113,38 +166,10 @@ def get_participants_emails_by_event(db: Session, event_id: int) -> List[str]:
     """Returns a list of all the email address of the event invited users,
         by event id."""
 
-    return [email[0] for email in db.query(User.email).
+    return (
+        [email[0] for email in db.query(User.email).
             select_from(Event).
```

Comment:
תהיה: אולי שווה לפתוח על זה טבלה בDB?

Reply:
@sagizaidor I'm not actually sure, this isn't my code, but lint check failed me for it so I had to fix

Reply:
לכל משתמש יש אימייל ויש טבלה מקשרת בין האירוע למשתמשים. איזה טבלה אתה מציע להוסיף?